dojson/contrib/marc21/__init__.py

```diff
@@ -7,6 +7,10 @@
 # modify it under the terms of the Revised BSD License; see LICENSE
 # file for more details.
 
-from model import marc21
+"""MARC standards based on `www.loc.gov/marc/ <http://www.loc.gov/marc/>`_."""
 
-__all__ = ('marc21',)
+from __future__ import absolute_import
+
+from .model import marc21, marc21_holdings
```

Comment:
should we go for the same naming convention for all of them `marc21_bibliographic`, `marc21_holdings`, `marc21_authority`, etc.?